contracts/staking/stakeManager/StakeManagerProxy.sol

```diff
@@ -0,0 +1,28 @@
+pragma solidity ^0.5.2;
+
+import {IGovernance} from "../../common/governance/IGovernance.sol";
+import {StakeManagerStorage} from "./StakeManagerStorage.sol";
+import {Proxy} from "../../common/misc/Proxy.sol";
+import {Registry} from "../../common/Registry.sol";
+import {StakingInfo} from "../StakingInfo.sol";
+import {StakingNFT} from "./StakingNFT.sol";
+import "../validatorShare/ValidatorShareFactory.sol";
+
+contract StakeManagerProxy is Proxy, StakeManagerStorage {
+    constructor(
+        address _proxyTo,
+        address _registry,
+        address _rootchain,
+        address _NFTContract,
+        address _stakingLogger,
+        address _validatorShareFactory,
+        address _governance
+    ) public Proxy(_proxyTo) {
```

Comment:
`public Proxy(_proxyTo) Lockable(_governance)` because StakeManagerStorage inherits from Lockable.

Reply:
forgot that I can call 2 `constructor`, due to this one fixed above comment as well. Thanks 😄